Wrappers/Python/cil/utilities/quality_measures.py

```diff
@@ -18,36 +18,90 @@
 # CIL Developers, listed at: https://github.com/TomographicImaging/CIL/blob/master/NOTICE.txt
 
 from cil.optimisation.functions import L2NormSquared, L1Norm
+from cil.framework import DataContainer
 import numpy as np
 
 
-def mse(dc1, dc2):    
+
+def mse(dc1, dc2, mask=None):    
     
     ''' Returns the Mean Squared error of two DataContainers
-    '''
     
-    diff = dc1 - dc2    
-    return L2NormSquared().__call__(diff)/dc1.size
+    Parameters
```

Comment:
Doc string should have a `Return`. and the description should be `Calculates the mean squared error of two images`

Consider adding the equation?